Scripts/script-ADGetUser.yml

```diff
@@ -302,6 +309,8 @@ outputs:
   description: The indicator's score
   type: number
 scripttarget: 0
+runonce: false
 dependson:
   must:
   - ad-search
+releaseNotes: "add escaping to filter"
```

Comment:
Release notes should start with capital and should be explain better the fix (users will not understand this..)